frontend/src/v5/ui/components/dashboard/dashboardList/dashboardListItem/shareModal/shareModal.component.tsx

```diff
@@ -21,40 +21,43 @@ import { FormModal } from '@controls/modal/formModal/formDialog.component';
 import { formatMessage } from '@/v5/services/intl';
 import { FormattedMessage } from 'react-intl';
 import { viewerShareLink } from '@/v5/services/routing/routing';
-import { IContainer } from '@/v5/store/containers/containers.types';
 import { useParams } from 'react-router-dom';
 import { ShareTextField } from '@controls/shareTextField';
 import { MailToButton } from './shareModal.styles';
 
 type IShareModal = {
 	openState: boolean;
-	container: IContainer;
+	title: string;
+	elementId: string;
+	elementName: string;
```

Comment:
I'm not sure about 'element' seems a bit generic. We could have something like 
```suggestion
   containerOrFederation: IContainer  | IFederation
```

Because viewerShareLink only works with federations of containers, I don't think this modal should handle more abstract things (for now, unless we want to share another type of urls like this)

Reply:
Done